### DISCUSSION TITLE
HTTPS via Nginx, how to disable keycloak https?

### DISCUSSION THREAD

**Discussion by luochen1990** [keycloak/keycloak#32263] [Q&A]
I already did everything needed for https tls support via nginx, and doen't want to touch them again, so how to tell keycloak to use http only and disable https?

**Comment by shawkins** [comment]
http-enabled=true - see https://www.keycloak.org/server/all-config?q=http-enabled

**Reply by luochen1990** [reply]
Is it possible to set something like `https-enabled=false` ?
